docs/development/Offline_mode.md

```diff
@@ -0,0 +1,76 @@
+# How to Implement Offline Capabilities
+
+**Flutter resources:**    
+https://github.com/tekartik/sqflite/blob/master/sqflite/README.md  
+https://pub.dev/packages/connectivity   
+https://flutter.dev/docs/cookbook/images/cached-images   
+https://pub.dev/packages/cached_network_image
+
+## Our solution suggestion
+We propose what is called an offline first approach because if Giraf wants to be a widely used app on different app markets, it is expected to be highly functional without internet connection. Offline first also consumes less battery and data - where especially long battery time is important for institutions and families going on trips.
+
+![image](https://cdn.discordapp.com/attachments/674940529124966402/691622267402190858/unknown.png)

```

Comment:
The image is a little misleading; When the memory cache and local database can't respond to a query, and the REST API returns an answer, the path back to the bloc from the model adapter (in the lower right corner) is unclear. This was mentioned in the feedback by group 15.